include/aspect/material_model/interface.h

```diff
@@ -222,6 +222,25 @@ namespace aspect
       std::vector<double> viscosities;
 
       /**
+       * Constitutive tensors at the given positions. This variable
+       * can be used to implement exotic rheologies such as anisotropic
+       * viscosity.
+       *
+       * @note The strain rate term in equation (1) of the manual will be
+       * multiplied by this tensor *and* the viscosity scalar (\\eta). If no
+       * value is assigned here the identity tensor will be used by default.
```

Comment:
Can you explain what "identity tensor" here actually means? I suppose you mean the rank-4 tensor `I` that when multiplied by a symmetric rank-2 tensor `epsilon` has the property that `I epsilon = epsilon`. That is the case for `I_{ijkl} = \delta_{ik}\delta_{jl}`.